### PR TITLE
Add charting to fluent 1 docsite

### PR DESCRIPTION
**Pull request by AtishayMsft**
## New Behavior
This PR contains following changes to enable charting controls in fluent 1 docsite.

- Adding pages for charting in docsite resources. 
- Updating  monaco editor supported packages to support Export to codepen button for charting controls.
- Disable SSR test for charts. Charts are not designed to render in server side

All the doc content has been reviewed by PM, designer, developers from the charting org.

Docsite PR updated with charting controls [here](https://fluentuipr.z22.web.core.windows.net/pull/28166/public-docsite/index.html#/controls/web/areachart)

## Related Issue(s)

- Fixes #

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/e101115deef04a486083afb56af060c03d9fa9f3'>e101115deef04a486083afb56af060c03d9fa9f3</a></sub>
<div id="bundleSizeComment9423"/>

File: apps/public-docsite-resources/config/api-docs.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ module.exports = {
   apiJsonPaths: [
```
Comment (fabricteam) [inline apps/public-docsite-resources/config/api-docs.js:12]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28166&commitId=049fb018621e578a9b327b89f0d844043d745991&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-AreaChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Custom Accessibility.default.chromium.png|11| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: apps/public-docsite-resources/config/api-docs.js (hunk 1 above)
Comment (fabricteam) [inline apps/public-docsite-resources/config/api-docs.js:12]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28166&commitId=7088fa3f0de7dd822dab532e0d7f49779943be27&env=prod&clientType=FLUENTUI)


  <details><summary><b>Pivot - Overflow</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: apps/public-docsite-resources/config/api-docs.js (hunk 1 above)
Comment (fabricteam) [inline apps/public-docsite-resources/config/api-docs.js:12]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28166&commitId=aa79dcd42bbb40ca1f0b7075e941d0eca2e4e4ed&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-AreaChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Custom Accessibility.default.chromium.png|11| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.default.chromium.png|1| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: apps/public-docsite-resources/config/api-docs.js (hunk 1 above)
Comment (fabricteam) [inline apps/public-docsite-resources/config/api-docs.js:12]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28166&commitId=7bc40f822df5c0775c792d5fc3ce914ea0c8b601&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-AreaChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Custom Accessibility.default.chromium.png|11| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: apps/public-docsite-resources/config/api-docs.js (hunk 1 above)
Comment (fabricteam) [inline apps/public-docsite-resources/config/api-docs.js:12]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28166&commitId=f1925a63784bcb8695d58768bd433b475015b75b&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-AreaChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Custom Accessibility.default.chromium.png|11| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: apps/public-docsite-resources/config/api-docs.js (hunk 1 above)
Comment (fabricteam) [inline apps/public-docsite-resources/config/api-docs.js:12]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28166&commitId=a4e2bf14e660364978fec47a1e74727bd0b3a232&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-AreaChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Custom Accessibility.default.chromium.png|11| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.default.chromium.png|1| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

Comment (fabricteam) [inline apps/public-docsite-resources/config/api-docs.js:12]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28166&commitId=1155d1f2a16d9fba3141be358569ef6a972aaeff&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-AreaChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Custom Accessibility.default.chromium.png|11| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.default.chromium.png|1| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: apps/public-docsite-resources/config/api-docs.js (hunk 1 above)
Comment (fabricteam) [inline apps/public-docsite-resources/config/api-docs.js:12]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28166&commitId=e243ead5914b4a47109fbcd3bf1492eb4c6545ac&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.default.chromium.png|1| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: apps/public-docsite-resources/config/api-docs.js (hunk 1 above)
Comment (fabricteam) [inline apps/public-docsite-resources/config/api-docs.js:12]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28166&commitId=ec7826126dac67f01859c45f81570c7faccea819&env=prod&clientType=FLUENTUI)


  <details><summary><b>ActivityItem</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ActivityItem.Personas - RTL.default.chromium.png|0| Removed|
|ActivityItem.Personas.default.chromium.png|0| Removed|
|ActivityItem.Root - RTL.default.chromium.png|0| Removed|
|ActivityItem.Root.default.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: apps/public-docsite-resources/config/api-docs.js (hunk 1 above)
Comment (fabricteam) [inline apps/public-docsite-resources/config/api-docs.js:12]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28166&commitId=71c889914cb0aa330f7cd7f085f4747d9bd1bb53&env=prod&clientType=FLUENTUI)


  <details><summary><b>Pivot - Overflow</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: packages/react-monaco-editor/src/utilities/defaultSupportedPackages.ts [hunk 2]
```diff
@@ -79,7 +79,7 @@ if (typesContext) {
   hooksGroup.packages.push({ packageName: '@fluentui/react-hooks', loadTypes });
   exampleDataGroup.packages.push({ packageName: '@fluentui/example-data', loadTypes });
 }
-if (typeof window === 'object' && window.location.href.indexOf('react-charting') !== -1) {
+if (typeof window === 'object' && window.location.href.indexOf('chart') !== -1) {
```
Comment (Hotell) [inline packages/react-monaco-editor/src/utilities/defaultSupportedPackages.ts:82]: can we add docs why are we searching for `chart` in particular ?

Comment (AtishayMsft) [reply packages/react-monaco-editor/src/utilities/defaultSupportedPackages.ts:85]: Added. Thanks.

**Review by Hotell — APPROVED** [review]

**Review by sopranopillow — APPROVED** [review]

**Review by krkshitij — APPROVED** [review]

**Comment by JustSlone** [comment]
🎉 thanks @AtishayMsft glad to see this merged!

**Pull request merged**